demo_formats.py

```
@@ -0,0 +1,78 @@
+    
+    <pre><code>
+# Sample Python code
+from markdown_lab_rs import convert_html, OutputFormat
+
+result = convert_html(html_content, url, OutputFormat.JSON)
+    </code></pre>
+</body>
+</html>
+"""
+
+def main():
```

Ruff (0.8.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)